install/traf_tools_setup.sh

```diff
@@ -62,9 +62,43 @@ function Usage {
    echo
 }
 
-  VERBOSE=0
-  TOOLSDIR=
-  BASEDIR=
+# -----------------------------------------------------------------------------
+# function: downloadSource - downloads and un-tars the requested file
+#    $1 - tar file to download 
+#    $2 - directory where source is untarred
+# -----------------------------------------------------------------------------
+function downloadSource
+{
+  # currently only tar files ending in "tar.gz" and "tgz" are recognized
+  TARSUFFIX="tar.gz"
+  if [[ ! $1 == *$"$TARSUFFIX" ]]; then
+    TARSUFFIX="tgz"
+  fi
+
+  if [ ! -e $BASEDIR/$2.$TARSUFFIX ]; then
+    wget $1  >>$LOGFILE 2>&1
```

Comment:
That is a good suggestion.  I would like to make other improvements to this script and will make a note of this issue.